### PR TITLE
Add pprof for debug

### PR DESCRIPTION
**Pull request by 3pointer**
### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has exported function/method change
 - Has exported variable/fields change
 - Has interface methods change
 - Has persistent data change

Side effects

 - Possible performance regression
 - Increased code complexity
 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation
 - Need to be included in the release note

### CODE REVIEW

File: sync_diff_inspector/main.go [hunk 1]
```diff
@@ -104,3 +106,12 @@ func checkSyncState(ctx context.Context, cfg *config.Config) bool {
 	}
 	return d.PrintSummary(ctx, &cfg.Task)
 }
+
+func startPProf(cfg *config.Config) error {
+	// Initialize the pprof server.
+	if cfg.StatusAddr != "" {
+		return sync_utils.StartPProfListener(cfg.StatusAddr)
+	}
+	sync_utils.StartDynamicPProfListener()
```
Comment (glorv) [inline sync_diff_inspector/main.go:115]: I perfer to provide a default status-addr instead of dynamic address

Comment (3pointer) [reply sync_diff_inspector/main.go:115]: default status-addr may conflict...

**Comment by dveeden** [comment]
@3pointer what's the plan with this PR?

**Comment by 3pointer** [comment]
It's a debug option. let's close it until someone need this feature.

**Pull request closed without merging**